<launch>
  <arg name="cam_config" default="$(find stargazer_ros_tool)/param/cam.yaml"/>
  <arg name="map_config" default="$(find stargazer_ros_tool)/param/map.yaml"/>
  <arg name="bag_file" default="$(find stargazer_ros_tool)/res/Stargazer.bag"/>

  <node pkg="stargazer_ros_tool" type="reprojection_visualizer" name="reprojection_visualizer" output="screen"
        required="true">
    <param name="cam_config" value="$(arg cam_config)"/>
    <param name="map_config" value="$(arg map_config)"/>
    <param name="bag_file" value="$(arg bag_file)"/>
    <param name="landmark_topic" value="/landmarks_seen"/>
    <param name="pose_topic" value="/stargazer/camera_pose"/>
    <param name="img_topic" value="/image_undistorted"/>
    <param name="waitTime" value="10"/>
  </node>

</launch>
